rplidar.py

```diff
@@ -156,7 +156,7 @@ def stop_motor(self):
         '''Stops sensor motor'''
         self.logger.info('Stoping motor')
         # For A1
-        self._serial_port.setDTR(True)
+        #self._serial_port.setDTR(True)  #here causing conflict with A2. DTR has to be always False to communicate.
         # For A2
         self.set_pwm(0)
```

Comment:
yes. good idea also. but, although it is not necessary, user will not able to use set_pwm(duty_cycle) to start motor after they used stop_motor() to stop the motor?